zebra-consensus/src/transaction.rs

```diff
@@ -192,7 +193,25 @@ where
                         // correctly.
 
                         // Then, pass those items to self.joinsplit to verify them.
-                        check::validate_joinsplit_sig(joinsplit_data, shielded_sighash.as_bytes())?;
+                        // check::validate_joinsplit_sig(joinsplit_data, shielded_sighash.as_bytes())?;
```

Comment:
```suggestion
```
As the function `validate_joinsplit_sig` was removed from the codebase, lets remove the comment.